### ISSUE TITLE
Show the auth code

### ISSUE THREAD

**Issue by nikmd23** [lostintangent/azjs#21]
In azjs `0.1.13`, the browser based interactive login prompt no longer shows the auth code needed, it only copies it into the clipboard.

This is a really nice experience provided the *default* browser on your machine is the one you want to auth with, in my case it isn't. So here was my experience:

- Type in `azjs deploy`.
- Chrome pops open and navigates to the login page.
- I want to use Edge, because that's the browser that is already 2FA'ed for me.
- Now I have to open Notepad, paste the code in my clipboard to it, switch back to Chrome, copy the login page url, open Edge, paste the login page url, switch to Notepad, copy the previously pasted auth code, switch back to Edge, paste the auth code. 😞 

Basically, it boils down to the fact that I need two difficult to type strings (auth code and URL), but neither of them are shown in my terminal, so I have to dance a bit if the default browser isn't the one I want to use.

**Comment by lostintangent** [comment]
Yeah that's good feedback. I'll add the code back and will also display the URL to auth, so it allows more control.

**Comment by lostintangent** [comment]
@nikmd23 I added the code and URL back to the default message, so when you update the `azjs` bits to test out the hang issue, you should see this change as well. Let me know what you think!

**Comment by lostintangent** [comment]
Closing this as resolved since the fixed is now published, but please re-open if you have any other feedback!

**Comment by nikmd23** [comment]
👌
